lib/trackler/implementation.rb

```diff
@@ -56,6 +55,8 @@ def git_url
 
     private
 
+    attr_reader :track
```

Comment:
This is safe, nobody started using it since it was added on 2017-04-19 (~3 days ago)